packages/block-editor/src/components/list-view/use-block-selection.js

```diff
@@ -132,14 +132,18 @@ export default function useBlockSelection() {
 				if ( title ) {
 					label = sprintf(
 						/* translators: %s: block name */
-						__( '%s deselected.' ),
+						__( '"%s" block deselected.' ),
 						title
 					);
 				}
 			} else if ( selectionDiff.length > 1 ) {
 				label = sprintf(
-					/* translators: %s: number of deselected blocks */
-					__( '%s blocks deselected.' ),
+					/* translators: %d: number of deselected blocks */
+					_n(
+						'%d block deselected.',
+						'%d blocks deselected.',
+						selectionDiff.length
+					),
```

Comment:
Even though this only runs when there are more than 1 items, We should use `_n` because in some languages (for example Russian) 11, 21, 31 etc are singular while 5 is plural etc.